scripts/DllManipulator.cs

```diff
@@ -644,34 +654,34 @@ private static void WriteNativeCrashLog(NativeFunction nativeFunction, object[]
 
         private static IntPtr SysLoadDll(string filepath)
         {
-#if UNITY_STANDALONE_WIN
-            return PInvokes_Windows.LoadLibrary(filepath);
-#elif UNITY_STANDALONE_LINUX
+#if UNITY_STANDALONE_LINUX
             return PInvokes_Linux.dlopen(filepath, (int)Options.posixDlopenFlags);
 #elif UNITY_STANDALONE_OSX
             return PInvokes_Osx.dlopen(filepath, (int)Options.posixDlopenFlags);
+#else // UNITY_STANDALONE_WIN
```

Comment:
Is the way it was somehow bad? I understand the addition of `|| UNITY_EDITOR_XXX` (which should also go there), but not the commented else. I mean, if for any reason the running platform happens not to be listed here, the code should not even compile.

Reply:
I though it might be useful in some cases as there's [quite a lot of platforms](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html). But then it probably makes most sense to leave it as `#elif UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN` as this is what is directly supported. I will change it to that.